numpy/__init__.py

```diff
@@ -247,7 +247,7 @@ def __getattr__(attr):
             except KeyError:
                 pass
             else:
-                warnings.warn(msg, RuntimeWarning)
+                warnings.warn(msg, DeprecationWarning)
```

Comment:
An orthogonal fix:
```suggestion
                warnings.warn(msg, RuntimeWarning, stacklevel=2)
```

Reply:
This would make it clear that an old astropy install is at fault, which is maybe enough for us not to worry about the kids warnings.

Reply:
Right, I hadn't noticed the missing stacklevel (coincidentally, I think this is about the only file the tests don't scan for this).